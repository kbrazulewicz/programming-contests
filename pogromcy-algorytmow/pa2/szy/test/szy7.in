7
40
245540
255576
260571
521144
225535
270594
250550
125265
255561
255562
365813
275605
230526
456001
260592
240528
420924
526155
285627
225595
340748
245559
225496
621364
435977
265583
215473
450990
786747
225495
826815
721584
230506
526175
245549
245539
225499
927035
295658
295649
5176665