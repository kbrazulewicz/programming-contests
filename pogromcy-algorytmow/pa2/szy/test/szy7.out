1100100001101010100101101000100001101000
